api/stats.js

```diff
@@ -1,31 +1,73 @@
 import { config } from './../config.js';
 import { SnowflakeUtil, MessageEmbed } from "discord.js";
 
+const removeParentheses = word => word.replace(/\(|\)/g, '');
 export class Stats {
```

Comment:
I know this is not part of this commit but the use of class components and the "this" object is many times considered an anti pattern, I suggest to use global variables and a functional paradigm instead.

Reply:
can you expand on why using classes is an anti-pattern? why using global variables is great?

Reply:
@TomasGonzalez it's better if we discuss that. It might be that we don't know about it.